app/views/posts/index.html.erb

```diff
@@ -1 +1,25 @@
-<h1>Here is a list of posts for all user</h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <style>
+    </style>
+  </head>
+  <body>
+   <h1>Here is the list of Posts<h1/>
+    <ul>
+    <li>
+    <% @posts.each do |post| %>
+      <a href='posts/<%= post.id %>'>
+      <%= post.text %>
+      </a>
+    </li>
+    <p>Comments:
+    <%= post.commentscounter %>
+      </p>
+      <p>likes: 3
+      </p>
+      <% end %>
+    </ul>
+  </body>
+</html>
```

Comment:
- [ ] Kindly ensure the Post index page matched the wireframe.

  Your Implementation | Wireframe
  :---:|:---:
  ![liee](https://user-images.githubusercontent.com/36542986/225569876-7a5942f8-1453-4e9e-9705-2e0d4380351b.jpg) | ![oeye](https://user-images.githubusercontent.com/36542986/225569892-34cef474-18ee-401c-8d7c-cce0b7c2309d.jpg)